corehq/apps/locations/permissions.py

```diff
@@ -103,11 +104,16 @@
 
 
 def locations_access_required(view_fn):
-    """
-    Decorator controlling domain-level access to locations.
-    """
-    return login_and_domain_required(
+    """Decorator controlling domain-level access to locations features."""
+    return (login_and_domain_required(
         requires_privilege_raise404(privileges.LOCATIONS)(view_fn)
+    ))
+
+
+def require_can_edit_locations(view_fn):
```

Comment:
looking at the symmetry between this and `locations_access_required`. First thought is is it too late to change `locations_access_required` to `require_can_view_locations`?

It's not immediately clear to me if the word _access_ implies _view_ or _edit_ permissions (although I would guess _view_ given the presence of `require_can_edit_locations`).